apitest/src/main/java/bisq/apitest/config/BisqAppConfig.java

```diff
@@ -0,0 +1,123 @@
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.apitest.config;
+
+import bisq.seednode.SeedNodeMain;
+
+import bisq.desktop.app.BisqAppMain;
+
+
+
+import bisq.daemon.app.BisqDaemonMain;
+
+/**
+ Some non user configurable Bisq seednode, arb node, bob and alice daemon option values.
+ @see <a href="https://github.com/bisq-network/bisq/blob/master/docs/dev-setup.md">dev-setup.md</a>
+ @see <a href="https://github.com/bisq-network/bisq/blob/master/docs/dao-setup.md">dao-setup.md</a>
+ */
+@SuppressWarnings("unused")
+public enum BisqAppConfig {
+
+    seednode("bisq-BTC_REGTEST_Seed_2002",
+            "bisq-seednode",
+            "\"-XX:MaxRAM=2g -Dlogback.configurationFile=apitest/build/resources/main/logback.xml\"",
+            SeedNodeMain.class.getName(),
+            2002,
+            5120,
+            -1),
+    arbdaemon("bisq-BTC_REGTEST_Arb_dao",
+            "bisq-daemon",
+            "\"-XX:MaxRAM=2g -Dlogback.configurationFile=apitest/build/resources/main/logback.xml\"",
+            BisqDaemonMain.class.getName(),
+            4444,
+            5121,
+            9997),
+    arbdesktop("bisq-BTC_REGTEST_Arb_dao",
+            "bisq-desktop",
+            "\"-XX:MaxRAM=3g -Dlogback.configurationFile=apitest/build/resources/main/logback.xml\"",
+            BisqAppMain.class.getName(),
+            4444,
+            5121,
+            -1),
+    alicedaemon("bisq-BTC_REGTEST_Alice_dao",
+            "bisq-daemon",
+            "\"-XX:MaxRAM=2g -Dlogback.configurationFile=apitest/build/resources/main/logback.xml\"",
+            BisqDaemonMain.class.getName(),
+            7777,
+            5122,
+            9998),
+    alicedesktop("bisq-BTC_REGTEST_Alice_dao",
+            "bisq-desktop",
+            "\"-XX:MaxRAM=4g -Dlogback.configurationFile=apitest/build/resources/main/logback.xml\"",
+            BisqAppMain.class.getName(),
+            7777,
+            5122,
+            -1),
+    bobdaemon("bisq-BTC_REGTEST_Bob_dao",
+            "bisq-daemon",
+            "\"-XX:MaxRAM=2g -Dlogback.configurationFile=apitest/build/resources/main/logback.xml\"",
+            BisqDaemonMain.class.getName(),
+            8888,
+            5123,
+            9999),
+    bobdesktop("bisq-BTC_REGTEST_Bob_dao",
+            "bisq-desktop",
+            "\"-XX:MaxRAM=4g -Dlogback.configurationFile=apitest/build/resources/main/logback.xml\"",
+            BisqAppMain.class.getName(),
+            8888,
+            5123,
+            -1);
```

Comment:
How do these MaxRAM settings relate to our discussion about similar settings in build.gradle?

Reply:
Limiting RAM for `bisq-apitest` was not necessary -- and removed from the `bisq-apitest` script -- because it is a headless app.  But we need to limit RAM for `:desktop` apps started by `bisq-apitest`, like we do for the Linux Bisq UI.

Above, you mentioned:
>> Is this to do with bisq using too much RAM in some cases? But wasn't that only relevant for long-running processes?

The Bisq UI is using too much without this limit, especially when starting it on an empty datadir.  This is why `BisqAppConfig` has a larger limit for `bobdesktop` than `bobdaemon`.  It's not really necessary for daemons at this stage, but it might be useful for profiling long running tests in the future.